BUILD

```diff
@@ -8,7 +9,7 @@ objc_library(
     includes = ["Sources/ObjcSupport/include"]
 )
 
-# PodToBUILD is a core library enabling Skylark code generation
+# PodToBUILD is a core library enabling Starlark code generation
```

Comment:
Skylark was renamed to "Starlark" in August 2018 https://blog.bazel.build/2018/08/17/starlark.html